app/src/main/java/com/usepace/android/messagingcenter/clients/connection_client/SendBirdClient.java

```diff
@@ -65,13 +160,22 @@ public void onRegistered(SendBird.PushTokenRegistrationStatus status, SendBirdEx
                                                     if (e != null) {    // Error.
                                                         connectionInterface.onMessageCenterConnectionError(e.getCode(), new MessageCenterException(e.getMessage()));
                                                     } else {
-                                                        SendBird.disconnect(new SendBird.DisconnectHandler() {
```

Comment:
also this disconnect is needed for notifications

Reply:
done